sktime/utils/numba/tests/test_general.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""Tests for numba functiond."""
```

Comment:
```suggestion
"""Tests for numba functions."""
```

Reply:
I fixed one typo and created a new one :)